spec/shr_genetics.txt

```diff
@@ -13,68 +13,72 @@ CodeSystem:     UCUM = http://unitsofmeasure.org
+Description:	"Human Genome Variation Society (HGVS) nomenclature for a single DNA marker. The use of the nomenclature must be extended to describe non-variations (aka. wild types) see samples for wild type examples. (source: LOINC)"
+				FindingTopicCode is LNC#48004-6 "DNA change (c.HGVS)"
+				FindingResult value is type	string
 
 Element:		DNASequenceVariantId
-Concept:		LNC#48003-8 "DNA sequence variation identifier [Identifier]"
 Based on:		NonIndependentFinding
+Concept:		LNC#48003-8 "DNA sequence variation identifier [Identifier]"
 Description:	"A DNA Marker identifier conveys a universal or standard repository identifier for definitive characterstics of a DNA Marker. (recommend using NCBI dbSNP ids - rs#) (source: LOINC)"
 				FindingTopicCode is LNC#48003-8 "DNA sequence variation identifier [Identifier]"
-				FindingResult value is type string
+				FindingResult value is type	string
+
+Element:		DNASequenceVariantType
+Based on:		CodedNonIndependentFinding
+Concept:		LNC#48005-3 "DNA change type"
+Description:	"Codified type for associated DNA Marker. DNA Marker's use the HGVS notation which implies the DNA Marker Type, but the concurrent use of this code will allow a standard and explicit type for technical and display convenience."
+				FindingTopicCode is LNC#48005-3 "DNA change type"
+				FindingResult from DNAChangeTypeVS
 
-Element:		DNASequenceVariantName
-Concept:		LNC#48004-6 "DNA change (c.HGVS)"
-Based on:		NonIndependentFinding
-Description:	"Human Genome Variation Society (HGVS) nomenclature for a single DNA marker. The use of the nomenclature must be extended to describe non-variations (aka. wild types) see samples for wild type examples. (source: LOINC)"
-				FindingTopicCode is LNC#48004-6 "DNA change (c.HGVS)"
-				FindingResult value is type string
 
 EntryElement:	TumorMarker
 Based on:		CodedLaboratoryObservation
 Concept:		LNC#21924-6 "Tumor marker Cancer"
 Description:	"A substance found in tissue or blood or other body fluids that may be a sign of cancer or certain benign (noncancer) conditions. Most tumor markers are made by both normal cells and cancer cells, but they are made in larger amounts by cancer cells. A tumor marker may help to diagnose cancer, plan treatment, or find out how well treatment is working or if cancer has come back. Examples of tumor markers include CA-125 (in ovarian cancer), CA 15-3 (in breast cancer), CEA (in colon cancer), and PSA (in prostate cancer). (Source: NCI Diectionary).  Tumor markers differ from genetic markers in that they are measured at the levels of the protein and substance post-RNA protein synthesis."
-				FindingTopicCode from TumorMarkerVS // should we put TumorMarkerVS with the list of all possible tumor markers?
-//				FindingResult 	// the result is the name of the tumor marker.
+				FindingTopicCode is LNC#21924-6 "Tumor marker Cancer"
+				FindingResult from TumorMarkerVS // should we populate TumorMarkerVS with the list of all possible tumor markers?
```

Comment:
yes, this is already added in my latest commit to mlt_dev_biomarker.  Please check that first.